.gitignore

```diff
@@ -8,3 +8,81 @@
 /spec/reports/
 /tmp/
 /vendor/
+
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
```

Comment:
I think we'd rather just ignore the whole `.idea` folder

Reply:
Do you think I should do that here at all, or leave it to each dev to configure as @andyw8 suggested?

Reply:
(this is also what's recommended on https://git-scm.com/docs/gitignore#:~:text=Patterns%20which%20a,is%20used%20instead. )